easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.9.0-foss-2023b.eb

```diff
@@ -0,0 +1,37 @@
+# #
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+# #
+easyblock = 'ConfigureMake'
+
+name = 'arpack-ng'
+version = '3.9.0'
+
+homepage = 'https://github.com/opencollab/arpack-ng'
+description = "ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['24f2a2b259992d3c797d80f626878aa8e2ed5009d549dad57854bbcfb95e1ed0']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('Eigen', '3.4.0'),
+]
+
+preconfigopts = "sh bootstrap && "
+configopts = '--enable-mpi --with-pic --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+github_account = 'opencollab'
+
+sanity_check_paths = {
+    'files': ['lib64/libarpack.la', 'lib64/libarpack.so', 'lib64/libparpack.la', 'lib64/libparpack.so'],
```

Comment:
please replace `.so` with `'....%s % SHLIB_EXT'` (see previous versions of arpack)